packages/backend/server/schema.prisma

```diff
@@ -274,9 +285,14 @@ model Update {
   id          String   @map("guid") @db.VarChar
   blob        Bytes    @db.ByteA
   createdAt   DateTime @map("created_at") @db.Timestamptz(3)
+  // TODO(@darkskygit): fullfill old update, remove default value in next release
+  createdBy   String   @map("created_by") @db.VarChar @default("system")
+
+  // delete update if createor's account is deleted
+  createdByUser User @relation(name: "createdUpdate", fields: [createdBy], references: [id], onDelete: Cascade)
```

Comment:
updates should be kept because it might be broadcasted to other client and pending updates could rely on it